### PR TITLE
refactor(framework) Update CLI SSL flags

### PR DESCRIPTION
**Pull request by danielnugraha**
### Description

Having flag that expects multiple argument is confusing to users.

### Explanation

Update CLI SSL flag to multiple flags that each expects one argument and update the docs accordingly.

### Checklist

- [ ] Implement proposed change
- [ ] Write tests
- [ ] Update [documentation](https://flower.ai/docs/writing-documentation.html)
- [ ] Make CI checks pass
- [ ] Ping maintainers on [Slack](https://flower.ai/join-slack/) (channel `#contributions`)

### CODE REVIEW

File: src/py/flwr/server/app.py [hunk 1]
```diff
@@ -512,21 +515,51 @@ def _try_obtain_certificates(
...
+            return certificates
+        if args.ssl_certfile or args.ssl_keyfile:
+            sys.exit(
+                "You need to provide valid file paths to `--ssl-certfile` "
+                "and `--ssl-keyfile` to create a secure connection "
+                "in Fleet API server (REST, experimental)."
+            )
+    sys.exit(
+        "Certificates are required unless running in insecure mode. "
+        "Please provide certificate paths with '--certificates' or run the server "
```
Comment (Moep90) [inline src/py/flwr/server/app.py:560]: I assume the flag needs to be updated to `--ssl-certfile`?

**Review by Robert-Steiner — COMMENTED** [review]
We also need to update the `/doc/source/how-to-run-flower-using-docker.rst` and `/doc/source/how-to-upgrade-to-flower-next.rst` file. The `--certificates` flag also appears in the translations but I'm not sure if need to change them manually.

File: src/py/flwr/server/app.py (hunk 1 above)
Comment (danielnugraha) [reply src/py/flwr/server/app.py]: Resolved, I checked all --certificates occurrences and replace them accordingly.

**Review by tanertopal — APPROVED** [review]
lgtm

**Pull request merged**